lib/greed.rb

```diff
@@ -0,0 +1,12 @@
+absolute_path = File.expand_path(File.dirname(__FILE__))
+require absolute_path + '/greed/game'
+
+include Greed
```

Comment:
ruby's `include` keyword is for "mixing in" all the methods from the Module into the current class. . Here it will be including the `Greed` module onto the `main` object, which isn't something you want to do. It does allow you to refer to `Game` without specifying `Greed::Game`, but this is just a side effect.

Instead it would be better just to refer to `Greed::Game` below.